modules/frontend/metrics_query_handler.go

```diff
@@ -51,14 +51,17 @@ func newQueryInstantStreamingGRPCHandler(cfg Config, next pipeline.AsyncRoundTri
 		httpReq = httpReq.Clone(ctx)
 
 		var finalResponse *tempopb.QueryInstantResponse
-		c, err := combiner.NewTypedQueryRange(qr)
+		c, err := combiner.NewTypedQueryRange(qr, cfg.Metrics.Sharder.MaxResponseSeries)
 		if err != nil {
 			return err
 		}
 
 		collector := pipeline.NewGRPCCollector(next, cfg.ResponseConsumers, c, func(qrr *tempopb.QueryRangeResponse) error {
 			// Translate each diff into the instant version and send it
 			resp := translateQueryRangeToInstant(*qrr)
+			// series already limited by the query range combiner just need to copy the status and message
+			resp.Status = qrr.Status
```

Comment:
why don't we do this in translateQueryRangeToInstant? seems like we need similar logic in the http handler